cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamFileJanitor.java

```diff
@@ -54,8 +54,7 @@ public void cleanAll() throws IOException {
       return;
     }
     for (Location streamLocation : streamBaseLocation.list()) {
-      //TODO: construct namespace from streamLocation (Locations.parent.getname)
-      Id.Stream streamId = Id.Stream.from(Constants.DEFAULT_NAMESPACE, streamLocation.getName());
+      Id.Stream streamId = StreamUtils.getStreamIdFromLocation(streamLocation);
```

Comment:
That's how the StreamFileJanitor is designed - to clean up based only upon the file system, and not lookup anything in meta store or elsewhere.